--- Makefile.old	2005-03-13 19:14:40.524896600 +0100
+++ Makefile	2005-03-13 19:15:43.230363920 +0100
@@ -1066,13 +1066,13 @@
 #
 
 # Full path for directory that will hold the exception handler file:
-EXCEPTION_DIR = .
+EXCEPTION_DIR = /usr/share/spim
 
 # Full path for the directory that will hold the executable files:
-BIN_DIR = /usr/unsup/bin
+BIN_DIR = /usr/bin
 
 # Full path for the directory that will hold the man files:
-MAN_DIR = /var/unsup/man
+MAN_DIR = /usr/man
 
 # If you have flex, use it instead of lex.  If you use flex, define this
 # variable and set LEXFLAGS.
